k8s/overlays/main_versions/test/kustomization.yaml

```diff
@@ -39,79 +39,79 @@ images:
     newTag: 0.15.0
   - name: gridcapa-rao-runner
     newName: docker.io/farao/gridcapa-rao-runner-with-xpress
-    newTag: v1.21.1
+    newTag: v1.22.1
   - name: gridcapa-app
     newName: docker.io/farao/gridcapa-app
     newTag: v1.14.0
   - name: gridcapa-gateway
     newName: docker.io/farao/gridcapa-gateway
-    newTag: 1.2.0
+    newTag: 1.3.0
   - name: gridcapa-cse-input-data-bridge
     newName: docker.io/farao/gridcapa-data-bridge
-    newTag: v1.4.0
+    newTag: v1.5.0
   - name: gridcapa-cse-adapter
     newName: docker.io/farao/gridcapa-cse-adapter
-    newTag: v1.14.1
+    newTag: v1.15.0
   - name: gridcapa-cse-job-launcher
     newName: docker.io/farao/gridcapa-job-launcher
-    newTag: v1.10.0
+    newTag: v1.11.0
   - name: gridcapa-cse-rao-logs-dispatcher
     newName: docker.io/farao/gridcapa-rao-logs-dispatcher
-    newTag: v1.3.1
+    newTag: v1.4.0
   - name: gridcapa-cse-import-runner
     newName: docker.io/farao/gridcapa-cse-cc-import-runner
-    newTag: v1.25.1
+    newTag: v1.26.1
   - name: gridcapa-cse-export-runner
     newName: docker.io/farao/gridcapa-cse-cc-export-runner
-    newTag: v1.25.1
+    newTag: v1.26.1
   - name: gridcapa-cse-task-manager
     newName: docker.io/farao/gridcapa-task-manager
-    newTag: v1.26.0
+    newTag: v1.26.1
   - name: gridcapa-cse-exporter
     newName: docker.io/farao/gridcapa-export
-    newTag: v1.6.2
+    newTag: v1.7.0
   - name: gridcapa-swe-input-data-bridge
     newName: docker.io/farao/gridcapa-data-bridge
-    newTag: v1.4.0
+    newTag: v1.5.0
   - name: gridcapa-swe-exporter
     newName: docker.io/farao/gridcapa-export
-    newTag: v1.6.2
+    newTag: v1.7.0
   - name: gridcapa-swe-adapter
     newName: docker.io/farao/gridcapa-swe-adapter
-    newTag: v1.5.1
+    newTag: v1.6.1
   - name: gridcapa-swe-job-launcher
     newName: docker.io/farao/gridcapa-job-launcher
-    newTag: v1.10.0
+    newTag: v1.11.0
   - name: gridcapa-swe-rao-logs-dispatcher
     newName: docker.io/farao/gridcapa-rao-logs-dispatcher
-    newTag: v1.3.1
+    newTag: v1.4.0
   - name: gridcapa-swe-runner
     newName: docker.io/farao/gridcapa-swe-runner
-    newTag: v1.17.1.1
+    newTag: v1.17.3
```

Comment:
This version of swe-runner is based on the buggy powsybl-core v6.2.4 dependency. Wait for the next version before validating this PR.